README.md

```diff
@@ -0,0 +1,197 @@
+# Final group capstone - Book an Appointment
```

Comment:
- [ ] You have included some tests, so kindly add a section on your readme called "tests" with instructions about how to correctly run them.

👍 

![image](https://github.com/Salimer/Final-Capstone-Back-End/assets/57421823/fbe9ad33-b3fc-44c3-8342-9074db4d7f33)